### PR TITLE
Increment the version number.

### PR DESCRIPTION
**Pull request by rma-rripken**
Increment version string.  #248

### CODE REVIEW

File: cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java [hunk 1]
```diff
@@ -102,6 +102,7 @@ public class ApiServlet extends HttpServlet {
     public static final String OFFICE_ID = "office_id";
     public static final String DATA_SOURCE = "data_source";
     public static final String DATABASE = "database";
+    public static final String VERSION = "3.0";
```
Comment (MikeNeilson) [inline cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java]: why are we jumping to 3?

Comment (rma-rripken) [reply cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java]: It doesn't have to be 3.0.  2.1?

Comment (MikeNeilson) [reply cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java]: shove the gradle version property into a resource file and pull it from there?

Comment (rma-rripken) [reply cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java]: oh. 2.4 to match the gradle.

Comment (MikeNeilson) [reply cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java]: or at least the major/minor. There's a use case to know if it's def newer to run performance checks but I can add a hash laster.

**Review by MikeNeilson — APPROVED** [review]

**Comment by adamkorynta** [comment]
Still would like it to grab the version from gradle instead of duplicating....

**Comment by rma-rripken** [comment]
I thought about figuring out how to pull it from gradle or have the build put it in a resource and ... well, maybe next release?

**Comment by MikeNeilson** [comment]
agreed, not hard but this at least works for now.

**Pull request merged**